pkg/faketikv/cluster.go

```diff
@@ -28,18 +28,20 @@ import (
 // ClusterInfo records all cluster information.
 type ClusterInfo struct {
 	*core.RegionsInfo
-	Nodes map[uint64]*Node
+	Nodes  map[uint64]*Node
+	logger *log.Logger
```

Comment:
Can we use a global logger?

Reply:
Seems we will have dependency problem. Not sure which package to put the global logger in. Any suggestions for that?

Reply:
Maybe create a sub-package in faketikv?